src/Donors/Actions/LoadDonorsListTableAssets.php

```diff
@@ -38,6 +39,7 @@ public function __invoke()
             'adminUrl' => admin_url(),
             'pluginUrl' => GIVE_PLUGIN_URL,
             'dismissedRecommendations' => $this->getDismissedRecommendations(),
+            'recurringDonations' => Utils::isPluginActive('give-recurring/give-recurring.php'),
```

Comment:
When creating variables for booleans, it tends to be much more clear/intuitive when you use the _is_ / _enabled_  terminology otherwise you might think this is literally a variable for recurring donations 😏 

For example:
- `isRecurringEnabled`
- `recurringDonationsEnabled`

Reply:
Got it. I updated to the Donations Stas for consistency as well: f258d9d79f24e12a51399f33175b5c4047a0b5b7